app/models/collection.ts

```diff
@@ -47,7 +47,7 @@ export default class CollectionModel extends OsfModel {
 
     @computed(`{${choicesFields.join()}}.length`)
     get displayChoicesFields() {
-        return choicesFields.filter(field => !!this[field].length);
+        return choicesFields.filter(field => !!this[field].length );
```

Comment:
@Brian, I believe a space has been added here. If that is intentional, it is okay. If not, we may want to remove this.